src/shell.js

```diff
@@ -34,6 +34,8 @@ var /** @type {{
 }}
  */ Module;
 if (!Module) /** @suppress{checkTypes}*/Module = {"__EMSCRIPTEN_PRIVATE_MODULE_EXPORT_NAME_SUBSTITUTION__":1};
+#elif AUDIO_WORKLET
+var Module = globalThis.Module || (typeof {{{ EXPORT_NAME }}} != 'undefined' ? {{{ EXPORT_NAME }}} : {});
```

Comment:
Audio Worklets have a different model to Web Workers in that they are module-based, so code is not added from inside them with `importScripts()` like Web Workers, but the main thread injects new modules into them via [audioWorklet.addModule()](https://developer.mozilla.org/en-US/docs/Web/API/Worklet/addModule).

To access the actual global namespace from these added scripts, we must then use [globalThis.Module](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/globalThis) instead of being able to just write `Module`.

(I did experiment with a direction of maybe we could then use `globalThis.` unconditionally, but that ran into a depth of other issues with Node.js, so settled with this as the simplest form)